### DISCUSSION TITLE
Update Stackblitz PrimeNG Repro Template to v16

### DISCUSSION THREAD

**Discussion by psarno** [primefaces/community#94] [PrimeNG] [answered]
The official Primefaces - Primeng Issue Template is still using PrimeNG 14.0.2 and Angular 14.

I have tried to update this myself to the latest version but end up with errors such as:

> Menubar | ReferenceError: Cannot access 'Menubar' before initialization

This is with no code in the TS or anything in the template.

I have other bug reports open but I can't give a repro link as the Stackblitz template is out of date.

Can someone on the official team provide a new link to a v16 template?

![image](https://github.com/primefaces/community/assets/18640860/9179e114-2e01-428f-9744-74d9c70af7a0)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi, 

Thanks for reporting the issue, we've just updated it to v16: https://github.com/primefaces/primeng-issue-template
